docs/pages/bare/using-expo-client.md

```diff
@@ -1,49 +1,69 @@
+## Usage
 
-## Why might you want to do this?
+Inside any React Native app, start a [development server with Expo CLI](/workflow/expo-cli#develop):
 
-There are a number of benefits to keeping your project runnable in the Expo Go app.
+<Terminal cmd={[
+"$ npx expo start",
+]} cmdCopy="npx expo start" />
 
-- Share your progress with stakeholders by publishing or sharing the development URL to see changes live
-- Continuously deploy builds from pull requests
-- No need to do native builds for iOS and Android in development because you use the Expo Go app instead
-- Develop the JavaScript side of your app from any machine of your choice, eg: use Windows for iOS development if you have an iOS device
-- Easily get new contributors set up on the project, only Node.js and a phone are required
-- You can use [Expo CLI](/workflow/expo-cli) for a great development experience
+<BoxLink title={<>Don't have <InlineCode>npx expo start</InlineCode>?</>} href="/bare/installing-expo-modules" description={<>Install and configure the <InlineCode>expo</InlineCode> package in your project.</>} />
 
-## Practical patterns for client-compatible bare apps
+Now you can launch your app in Expo Go by pressing `i` or `a` in the Terminal UI. Some features may cause your app to throw errors because certain native code is missing, continue reading to learn how you can conditionally skip unsupported APIs.
 
-### Prefer `npx expo install` over `npm install` to add Expo SDK packages
+> Unlike `npx react-native start` the command `npx expo start` hosts an app manifest that dev clients like Expo Go can use to load arbitrary projects. Think of an app manifest like the `<head />` element of an `index.html` but for React Native apps. To view this manifest, visit the dev server URL in your web browser.
 
-This will ensure that you get a version of the package that is compatible with the SDK version in your app. If you use `npm install` directly instead, you may end up with a newer version of the package that isn't supported in Expo Go yet.
+## Installing libraries
 
-### Use conditional inline requires to provide fallbacks
+You'll want to ensure your app is using the most compatible library versions for the project's `react-native` version, this means using `npx expo install` instead of `npm install`. Learn more in [`npx expo install`](/workflow/expo-cli#install).
```

Comment:
```suggestion
Ensure your app uses the most compatible library versions for the project's `react-native` version. This means that use `npx expo install` instead of `npm install` to install libraries. Read more about [`npx expo install`](/workflow/expo-cli#install).
```